CMakeLists.txt

```diff
@@ -24,13 +24,17 @@ ENDIF(NOT CMAKE_BUILD_TYPE)
 
 message(STATUS "Build type CMAKE_BUILD_TYPE set to ${CMAKE_BUILD_TYPE}")
 
+include(UseCompilerCache)
+FIND_COMPILER_CACHE(PROGRAM ccache)
```

Comment:
Function call is upper case, upstream mandates lowercase for any kind of symbol and keyword, except variables.